src/main/java/net/fabricmc/accesswidener/EntryTriple.java

```diff
@@ -16,15 +16,23 @@
 
 package net.fabricmc.accesswidener;
 
-final class EntryTriple {
-	private final String owner;
-	private final String name;
-	private final String desc;
+public final class EntryTriple {
+	final String owner;
+	final String name;
+	final String desc;
+	final String fuzzy;
+	final boolean requiresSourceCompatibility;
```

Comment:
Im not a massive fan of this, its not really a simple `EntryTriple` anymore but now contains logic.

Reply:
It used to be a seperate class but since every usage in access-widener requires the fuzzy functionality I just merged the classes, maybe I can change the name?